### DISCUSSION TITLE
Kubernetes / Load Balancing Docs

### DISCUSSION THREAD

**Discussion by shubhamranjan** [bluenviron/mediamtx#804] [Questions]
Hi,

Thanks for such a great project and actively maintaining it as well.

I plan to deploy this server in a Kubernetes cluster and wanted to understand how the mechanism of load balancing w.r.t the following points.

- How will the reader end up on the same pod where the stream has been published ?
- What is the ideal way of load balancing publishers ? Round Robin as per whatever LB is being used or there is some integration available ?

Please feel free to point me to any docs if they already exists.

Thanks !

**Comment by aler9** [comment]
Moved here https://github.com/aler9/rtsp-simple-server/issues/998
